CVE-2024-9796/ubuntu1.yml

```diff
@@ -0,0 +1,161 @@
+- name: Install WordPress and MySQL
+  hosts: ubuntu1
+  become: yes
+  vars:
+    mysql_root_password: "supersecretrootpassword"
+    wordpress_db_name: "wordpress"
+    wordpress_db_user: "wordpress"
+    wordpress_db_password: "supersecretdbpassword"  
+    wp_site_title: "My WordPress Site for CVEX"
+    wp_admin_user: "HamsterAdmin" 
+    wp_admin_password: "supersecretadminpassword"
+    wp_admin_email: "calebee0462@gmail.com"
```

Comment:
Are you sure you want to leave your personal email here? Otherwise it looks awesome and it works.

Reply:
Yeah, I'm okay with leaving my email there. Thanks!